### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by mealilo** [KD7LRJ/3D-LDS-Temples#9] [General]
Thank you for this! Glad this could be hosted on GitHub, as Thingiverse is not stable. Tell Hugh thanks!

**Comment by OkinawJapanTemple** [comment]
Hey so I am a member of the church and living in Okinawa and the temple is compete and dedicated. So I was wondering if you could make a stl. file of the temple. There is a website released from the church that has a video of a 3d model, so if you could make it really exact like you normally do and maybe if you have time put the Okinawa Japan Temple on a Priority. Thanks if you can and thanks if you can't.
